### PR TITLE
Add pylama tests

### PR DESCRIPTION
**Pull request by cpaillet**
- Fix a typo on the travis file name..
- some others fix for pylama

### CODE REVIEW

File: cloudflare_exporter/collector.py [hunk 1]
```diff
@@ -89,10 +89,10 @@ def _get_cloudflare_analytics(token):
         # get the last 30 minutes (the very last 5 minutes are usually not correct)
         try:
             series = cloudflare.zones.analytics.colos(zone['id'], params={'since': -35, 'until': -5})
-        except CloudFlare.exceptions.CloudFlareAPIError as e:
-            code, _ = e.args
+        except CloudFlare.exceptions.CloudFlareAPIError as exception:
+            code, _ = exception.args  # pylint: disable=unbalanced-tuple-unpacking
             if code == 10_000:
```
Comment (dclaisse) [inline cloudflare_exporter/collector.py]: What about:
if exception.code, and get rid of args parsing?

**Review by dclaisse — CHANGES_REQUESTED** [review]

**Comment by dclaisse** [comment]
Sorry, I meant int(exception)

**Comment by cpaillet** [comment]
done

**Review by dclaisse — APPROVED** [review]
Nice!

**Pull request merged**